multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/util/DataSourceFactory.java

```diff
@@ -28,11 +28,29 @@ private HikariConfig createHikariConfig(CfJdbcService service, Integer maximumPo
         hikariConfig.setIdleTimeout(60000);
         hikariConfig.setMinimumIdle(10);
         hikariConfig.addDataSourceProperty("tcpKeepAlive", true);
+
+        configureSSLClientKeyIfExists(service, hikariConfig);
+
         if (maximumPoolSize != null) {
             hikariConfig.setMaximumPoolSize(maximumPoolSize);
         }
         hikariConfig.setRegisterMbeans(true);
         return hikariConfig;
     }
 
+    private void configureSSLClientKeyIfExists(CfJdbcService service, HikariConfig hikariConfig) {
+        Object clientKey = service.getCredentials()
+                                  .getMap()
+                                  .get("sslkey");
```

Comment:
Isn't this property a String, can you cast it?

Reply:
Yes, I assume there wouldn't be any problem with casting the property to String. I'll change it to that.

Reply:
+1